README.md

```diff
@@ -129,29 +195,27 @@ kubectl get managed
 
 #### Invite App Teams to you Organization in Upbound Cloud
 
-1. Create a team `Workspace` in Upbound Cloud, named `team1`.
-1. Enable self-service APIs in each `Workspace`.
-1. Invite app team members and grant access to `Workspaces` in one or more
-     `Platforms`.
+1. Create a Team `team1`.
+1. Invite app team members and grant access to `Control Planes` and `Repositories`.
 
 ### App Dev/Ops: Consume the infrastructure you need using kubectl
 
 #### Join your Organization in Upbound Cloud
 
 1. **Join** your [Upbound Cloud](https://cloud.upbound.io/register)
    `Organization`
-1. Verify access to your team `Workspaces`
+1. Verify access to your team `Control Planes` and Registries
 
 #### Provision a PostgreSQLInstance in your team Workspace GUI console
 
-1. Browse the available self-service APIs (XRDs) in your team `Workspace`
+1. Browse the available self-service APIs (XRDs) Control Plane
 1. Provision a `PostgreSQLInstance` using the custom generated GUI for your
```

Comment:
There is no resource available with the type `PostgreSQLInstance` after installing this platform configuration. It is the claim type for `CompositePostgreSQLInstance` and is not visible in the Control Plane console.

Reply:
Thanks! I will take a look